content/blog/2023/04/10/2023-04-10-jenkins-newsletter.adoc

```diff
@@ -0,0 +1,178 @@
+** Windows users expect their installers to be signed/secured (because of malwares and so on).
+The certificate has unfortunately expired a few days ago. Fortunately, Mark Waite and other members of the community managed to get a new one, so the latest weekly release is signed.
+** Lawyers had to be involved, the process is now complete
+** Jar file won’t be signed (low impact, low risk)
+** MSI installer is signed with new key
+* Latest updates on the agent images:
+** Ssh-agent link:https://github.com/jenkinsci/docker-ssh-agent/releases/tag/4.13.0[release 4.13.0]
+*** chore(deps): bump debian from bullseye-20230208 to bullseye-20230320 in /8/11/17bullseye (link:https://github.com/jenkinsci/docker-ssh-agent/pull/222[#222]) link:https://github.com/dependabot[@dependabot]
+** Docker-agent release link:https://github.com/jenkinsci/docker-agent/releases/tag/3107.v665000b_51092-6[3107.v665000b_51092-6]
+*** chore(deps): bump archlinux from base-20230226 to base-20230319.0.135218 in /11/archlinux (link:https://github.com/jenkinsci/docker-agent/pull/393[#393]) link:https://github.com/dependabot[@dependabot]
+*** chore(deps): bump debian from bullseye-20230227 to bullseye-20230320 in /11/17/bullseye (link:https://github.com/jenkinsci/docker-agent/pull/394[#394]) link:https://github.com/dependabot[@dependabot]
+* Experiments with RISC-V have link:/blog/2023/03/10/miniJen-and-RISC-V/[progressed].
+
+
+[[outreach]]
+image:/images/post-images/2023/01/12/jenkins-newsletter/outreach-and-advocacy.png[Outreach and advocacy Update]
+
+Contributed by: author:alyssat[Alyssa Tong]
+
+link:https://www.socallinuxexpo.org/scale/20x[*SCALE 20X#*]
+
+So thrilled to have been back at the usual spot (Pasadena Convention Center, CA) for SCALE this year, an added bonus were visits from special friends 🥰, Kohsuke Kawaguchi & Arun Gupta 🎉
+
+Many thanks to the Jenkins fans for stopping by the booth to let us know how much they love Jenkins!
+Special thanks to the SCALE committee for being a wonderful host! 🚀
+
+image:/images/post-images/2023/04/10/2023-04-10-jenkins-newsletter/image2.jpg[image,width=291,height=219]image:/images/post-images/2023/04/10/2023-04-10-jenkins-newsletter/image3.jpg[image,width=235,height=314]
+
+image:/images/post-images/2023/04/10/2023-04-10-jenkins-newsletter/image4.jpg[image,width=213,height=160]image:/images/post-images/2023/04/10/2023-04-10-jenkins-newsletter/image5.jpg[image,width=173,height=130]
+
+*Jenkins in Google Summer of Code (GSoC)*
+
+If you lurk on the Jenkins GSoC Gitter channel, you will be quite surprised at the level of engagement on this channel…it is anything but quiet.
+The hustle and bustle indicates the level of interest in Jenkins in GSoC.
+Here’s where we currently stand:
+
+* We’ve received 34 draft proposals for mentors to review
+* 16 proposals submitted via the Google Summer of Code portal.
+* Submissions are accepted until April 4 @ 1800 UTC.
```

Comment:
```suggestion
* We’ve received 34 draft proposals for mentors to review.
* 16 proposals have been submitted via the Google Summer of Code portal.
* Submissions are accepted until April 4 @ 1800 UTC.
```
should this information be updated due to the deadline being passed now? I think the total number of proposals is 55 valid submissions based on what @jmMeessen had shared.